lib/github-pages-health-check/domain.rb

```diff
@@ -204,7 +214,7 @@ def dns_zone_ns?
         end
       end
 
-      # Should the domain use an A record?
+      # Should the domain use an A or AAAA record?
```

Comment:
Is it safe to leave the method name this way given the change in scope?

Reply:
Yeah it's a great observation and I did ponder a little. I leaned on the notion that I _believe_ that A and AAAA records are serving the similar purposes, but I can look to separate those instead if I'm conflating them too much.